frontend/templates/product-list/sections/FilterableProductsSection/CurrentRefinements.tsx

```diff
@@ -11,11 +11,13 @@ import * as React from 'react';
 import {
    useClearRefinements,
    useCurrentRefinements,
+   useInstantSearch,
 } from 'react-instantsearch-hooks-web';
 
 export function CurrentRefinements() {
    const currentRefinements = useCurrentRefinements();
    const clearRefinements = useClearRefinements();
+   const { setIndexUiState } = useInstantSearch();
```

Comment:
This works, but I think we should use [`usePagination`](https://www.algolia.com/doc/api-reference/widgets/pagination/react-hooks/#hook) instead, as it is intended for setting the current page.

```suggestion
   const pagination = usePagination();
```

Reply:
⚠️ Yeah let's not use `useInstantSearch`, it has been the source of memory leaks and many issues. We should avoid it, and generally speaking it shouldn't be needed as it's a relatively low level hook.